model/EligibilityService.php

```diff
@@ -93,7 +93,7 @@ public function getAssignmentService() {
      * 
      * @param Resource $testCenter
      * @param Resource $delivery
-     * @return boolean
+     * @return Resource
```

Comment:
Is it safe to modify the return value? Maybe other extensions could depend on that.
Also in L101 it still returns a boolean value.

Reply:
I checked all places where we are using this function and we just check if returned value is not false, like:
```
if ($eligibilityService->createEligibility()) {
    // Do something
}
```
If we return created eligibility we don't have to get it from database each time we want to add new property (see methods removeEligibility(), getEligibleTestTakers(), setEligibleTestTakers() etc)